dbunit/README.md

```diff
@@ -0,0 +1,6 @@
+### Database schema
+
+![db schema](docs/db_schema.png)
```

Comment:
let's remove this since we will be adding the code to an existing module: libraries-testing